qa/logging-config/src/javaRestTest/resources/plugin-security.policy

```diff
@@ -0,0 +1,5 @@
+grant {
```

Comment:
Why is this needed when the test has security manager disabled?

Reply:
this is for the IntegTests.
the ESJsonLayoutTests which has securiyt manager disabled is a unit test